packages/gatsby-recipes/recipes/theme-ui.mdx

```diff
@@ -21,9 +22,9 @@ Add the plugin `gatsby-plugin-theme-ui` to your `gatsby-config.js`.
 
 Write out Theme UI configuration files.
 
-<GatsbyShadowFile
-  theme="gatsby-plugin-theme-ui"
-  path="src/index.js"
+<File
+  path="src/gatsby-plugin-theme-ui/index.js"
+  content="https://gist.github.com/KyleAMathews/ab59e200e4f8a1b4109dddb51b2140f9/raw/209a2e7c589766869522b12f7f6cecaf3f7a6f81/index.js"
```

Comment:
NB: this is a good pattern in the gist for extending `theme.styles` without affecting the base scales in the theme, but it could be simply `export default { ...funk }`

Reply:
hmmm which would you prefer?